odm2api/ODM2/services/readService.py

```diff
@@ -379,21 +385,25 @@ def getVariables(self, ids=None, codes=None, sitecode=None, results=False):
             return None
 
     # Method
-    def getMethods(self, ids=None, codes=None, type=None):
+    def getMethods(self, ids=None, codes=None, medtype=None, **kwargs):
```

Comment:
`medtype` seems like a misspelling, or else `med` doesn't seem like a good shorthand for "method". How about `mettype` or even better, full blown `methodtype`?

Reply:
I'll change to `methodtype`. This is the most clear.